src/components/quiz/Quiz.tsx

```
@@ -0,0 +1,244 @@
+                  <h1 className="text-2xl font-bold">
+                    Questão {currentStepIndex + 1}
+                  </h1>
+                  <p className="text-muted-foreground text-sm">
+                    {currentStepIndex + 1} de {quizData.questions.length}
+                  </p>
+                </div>
+
+                <div className="my-6">
+                  <div
+                    className="prose max-w-none"
+                    dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)